app/components/AddToCalendar/AddToCalendar.css

```diff
@@ -0,0 +1,13 @@
+.calendarToggleButton {
+  margin-top: 15px;
+  margin-bottom: 5px;
+}
+
+.calendarImportCard {
+  margin-top: 5px;
+}
+
+.calendarLink {
+  display: block;
+  margin-bottom: 15px;
+}
```

Comment:
Can we use `1rem` for `15px`  and `0.5rem` for `5px` instead? There's currently a lot of random spacing in the webapp